plugins/modules/site_workflow_manager.py

```diff
@@ -610,8 +630,29 @@ def site_exists(self):
         site_exists = False
         current_site = {}
 
-        if self.compare_dnac_versions(self.get_ccc_version(), "2.3.5.3") <= 0:
+        if self.compare_dnac_versions(self.get_ccc_version(), "2.3.7.6") >= 0:
+            sites = None
+            response = self.get_site(site_name_hierarchy)
```

Comment:
What happens if you pass None to site_name_hierarchy..Lets say the caller does not pass site_name_hierarchy..

Reply:
In this function "validate_site_input_data" used to validate the parent_name and name and these two are mandatory
Hence site_name_hierarchy can never be None/Empty

Reply:
if  site_name_hierarchy can never be None/Empty, then why are you making default param as None..

Reply:
In 2.3.5.3 version we are directly using site_exists function with no parameter, but in the latest version site_exits function with parameter.
So that we are using  site_name_hierarchy is None